syntax = "proto3";

package suggestions;

service SuggestionsService {
    rpc GetSuggestions (SuggestionsRequest) returns (SuggestionsResponse);
}

message SuggestionsRequest {
    string name = 1;
}

message SuggestionsResponse {
    repeated Suggestion suggestions = 1;
}

message Suggestion {
    int64 bookId= 1;
    string title = 2;
    string author = 3;
}
